packages/avro-cpp/buildinfo.json

```diff
@@ -0,0 +1,8 @@
+{
+  "requires": ["mesos", "boost-libs"],
+  "single_source": {
+    "kind": "url_extract",
+    "url": "http://apache.mirrors.pair.com/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz",
```

Comment:
I updated the line but this comment still shows the previous commits :/ Did github comments just become less awesome?